src/ga4gh/core/identifiers.py

```diff
@@ -122,7 +122,7 @@ def parse_ga4gh_identifier(ir):
         raise ValueError(ir) from e
 
 
-def ga4gh_identify(vro, in_place='default'):
+def ga4gh_identify(vro, in_place='default', as_version=None):
```

Comment:
Probably good to use an enum here if we are expected to support all previous versions (1.2, 1.1, 1.0) in future PRs. If not, we could probably just rename `as_version` to `to_1_3_version` or something and have it be a bool that defaults to `False`. Just so we're not hardcoding.

Reply:
If supporting multiple versions and using enum, it might be good to fail fast in the beginning of the function and raise an exception if version is not supported.

Reply:
agree–intent is to support multiple versions down the road and we should leave that door open. Also agree with fail fast approach.